crates/cli/src/main.rs

```diff
@@ -1,11 +1,21 @@
 use std::process::ExitCode;
 
 use clap::{Arg, Command};
-use mimalloc::MiMalloc;
 use spacetimedb_cli::*;
 use spacetimedb_paths::cli::CliTomlPath;
 use spacetimedb_paths::{RootDir, SpacetimePaths};
 
+#[cfg(not(target_env = "msvc"))]
+use tikv_jemallocator::Jemalloc;
+
+#[cfg(not(target_env = "msvc"))]
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
```

Comment:
so to confirm - this allocator change still somehow applies to `standalone` even though it's in a different binary? is that because we `exec` it?

Reply:
Nope. I thought it did, since I was still getting jemalloc stats, but it seems like the stat collection allocates a bit of memory even if we aren't using jemalloc anywhere else.